enos/enos-scenario-upgrade.hcl

```diff
@@ -281,7 +299,31 @@ scenario "upgrade" {
       quality.nomad_reschedule_alloc,
     ]
   }
- */
+
+  /*
+  step "run_servers_workloads" {
+   // ...
+  }
+
+  step "upgrade_client" {
+    description = <<-EOF
+    Upgrade the cluster's clients by invoking nomad-cc ...
+    EOF
+
+    module      = module.run_cc_nomad
+
+    verifies = [
+        quality.nomad_nodes_status,
+        quality.nomad_job_status
+    ]
+
+    variables {
+        cc_update_type = "client"
+        nomad_upgraded_binary             = step.copy_initial_binary.nomad_local_binary
+        // ...
+    }
+  }
+*/
```

Comment:
We've already defined `upgrade_client` above, so I think we can clean up this commented-out stuff.